### PR TITLE
incorporating automation framework in workflows

### PR DESCRIPTION
**Pull request by anunagula3**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -230,6 +249,12 @@ workflows:
           requires:
             - request-staging-approval
 
+      - acceptance-tests:
```
Comment (psoleckimoj) [inline .circleci/config.yml:252]: This is just an approval, it won't actually do anything. You need another task that requires this one and runs your acceptance tests job 🙂

Comment (anunagula3) [reply .circleci/config.yml]: just updated the script

File: .circleci/config.yml [hunk 2]
```diff
@@ -230,6 +249,18 @@ workflows:
...
 
+      - request-acceptance-tests-approval:
+          <<: *main_branch
+          type: approval
+          requires:
+            - deploy_staging
+
+      - acceptance_tests:
+          <<: *main_branch
+          env: "preprod"
```
[the remark this answers is not in the document]
Comment (anunagula3) [reply .circleci/config.yml]: No you're right just changed it

**Review by tpmcgowan — APPROVED** [review]
LGTM!

**Pull request merged**